packages/ui/src/components/commit-copy-actions.tsx

```diff
@@ -31,23 +31,24 @@ export const CommitCopyActions = ({
 
   return (
     <ShaBadge.Root>
-      <ShaBadge.Content className="p-0" asChild>
-        <button
-          className="size-full px-2.5 text-2 text-cn-foreground-3"
+      <ShaBadge.Content className="border-r" asChild>
+        <Button
```

Comment:
not a blocker for this PR, but if we have a specific and separate component for a ShaBadge, it should just take the sha as a string prop, and internally render the required button.

Reply:
Agree 💯

Reply:
Copy button should be built-in with `Sha.Content`